devenv.bat

```diff
@@ -54,6 +54,7 @@ SET CORECLR_ENABLE_PROFILING=1
 SET CORECLR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
 SET CORECLR_PROFILER_PATH=%~dp0\src\Datadog.Trace.ClrProfiler.Native\bin\%profiler_configuration%\%profiler_platform%\Datadog.Trace.ClrProfiler.Native.dll
 set DD_TRACE_CALLTARGET_ENABLED=0
+set DD_APPSEC_ENABLED=0
```

Comment:
Why add this here when it's off by default?

Reply:
just that my memory is bad, and I always have to look it up in our project to activate it here, so here just have to turn to 1^^  but can remove!